### PR TITLE
Rename core-header-panel to paper-header-panel

### PR DESCRIPTION
**Pull request by notwaldorf**
Also sorts out the bower.json config. @morethanreal, please take a look. Thanks!

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+    "polymer": "polymer/polymer#v0.8.0-rc.4",
+    "paper-styles": "PolymerElements/paper-styles#^0.8.3"
   },
   "devDependencies": {
-    "test-fixture": "Polymer/test-fixture#master",
-    "web-component-tester": "Polymer/web-component-tester#master"
+    "webcomponentsjs": "webcomponents/webcomponentsjs#~0.6.1",
+    "web-component-tester": "Polymer/web-component-tester#^2.2.3",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#master",
+    "test-fixture": "polymer/test-fixture#master"
```
Comment (morethanreal) [inline bower.json]: `^0.8.0`

File: demo/index.html [hunk 2]
```diff
@@ -11,10 +11,10 @@
 <html>
 
   <head>
-    <title>core-header-panel</title>
+    <title>paper-header-panel</title>
```
Comment (morethanreal) [inline demo/index.html]: paper-header-panel demo

File: index.html [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+        margin: 16px;
+      }
+    </style>
 
   </head>
-  <body unresolved>
+  <body>
 
-    <core-component-page></core-component-page>
+    <iron-doc-viewer src="paper-header-panel"></iron-doc-viewer>
```
Comment (morethanreal) [inline index.html]: does the doc page work? Or do you need `paper-header-panel.html`?

**Comment by morethanreal** [comment]
lgtm w/ nit

File: index.html (hunk 3 above)
Comment (notwaldorf) [inline index.html]: nope, that was my bad. thanks for catching it!

**Pull request merged**